pages/about.jsx

```diff
@@ -13,7 +13,7 @@ function About() {
       <section className="hero">
         <div className="container">
           <div className="text-wrapper w-full">
-            <h1 data-cy="pageTitle" className="title">
+            <h1 data-test-id="pageTitle" className="title">

```

Comment:
Change the `data-cy` attribute from Cypress to `data-test-id` attribute from Playwright